SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/ComponentTestUtils.h

```diff
@@ -24,6 +24,11 @@ inline ComponentData CreateTestComponentData(const Worker_ComponentId Id, const
 	return Data;
 }
 
+inline double GetValueFromSchemaComponentData(Schema_ComponentData* Data)
```

Comment:
Minor nit - I'd mildly prefer if this was something like GetValueFromTestComponentData to make it clear that this is all for test data.
But having typed that out that should really be made clear in some other way. Like a TestComponent class or something.

Reply:
So maybe this is a follow up

Reply:
This is another case of assuming test data includes a double value. I don't know if that's generally a good idea throughout the tests. There's definitely potential future clean up here, but for now I can change this to indicate that it's reading a test component data- maybe `GetDoubleValueFromTestComponentData`? Although double is already the return type so could be moot

Reply:
renamed to get value from test component data, and added clarification comment